solutions/tests/test_count_vowels.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from solutions.count_vowels import count_vowels
+
```

Comment:
if you can write here an docstring  explanation of the program what it does will be helpful

Reply:
will do right now